free_fleet_adapter/free_fleet_adapter/nav2_robot_adapter.py

```diff
@@ -51,11 +51,13 @@
 
 class Nav2TfHandler:
 
-    def __init__(self, robot_name, zenoh_session, tf_buffer, node):
+    def __init__(self, robot_name, zenoh_session, tf_buffer, node, robot_config_yaml=None, args=None):
```

Comment:
Since `Nav2TfHandler` does not require any more information than the name of the base link frame and map frame, we can just use two arguments `robot_frame='base_footprint'` and `map_frame='map'`, instead of passing any of these configs